src/libraries/Microsoft.PowerFx.Core/Functions/ITexlFunction.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal interface ITexlFunction
+    {
+        DPath Namespace { get; }
+
+        string Name { get; }
+
+        string LocaleInvariantName { get; }
```

Comment:
Can we avoid perpetuating the "LocaleInvariantName" for functions? Just as in #1013 , the function localization code is also unused.

Reply:
Removed interface